src/pytorch_lightning/loggers/tensorboard.py

```diff
@@ -97,13 +107,28 @@ def __init__(
         sub_dir: Optional[_PATH] = None,
         **kwargs: Any,
     ):
+        if not _TENSORBOARD_AVAILABLE and not _TENSORBOARDX_AVAILABLE:
```

Comment:
```suggestion
        if not _TENSORBOARD_AVAILABLE:
```
this is required depenency